utils/post_utils.jsx

```diff
@@ -327,16 +317,16 @@ export function makeCreateAriaLabelForPost() {
         (state, post) => getDisplayName(state, post.user_id),
         (state, post) => getReactionsForPost(state, post.id),
         (state, post) => get(state, Preferences.CATEGORY_FLAGGED_POST, post.id, null) != null,
-        (post, author, reactions, isFlagged) => {
-            return (intl) => createAriaLabelForPost(post, author, isFlagged, reactions, intl);
+        (state, post, author, reactions, isFlagged) => {
```

Comment:
This doesn't work because the state isn't passed to the final argument of `createSelector`, and we wouldn't want that anyway because of how selectors memoize their results (see https://developers.mattermost.com/contribute/redux/selectors/ for more info on that).

Instead, we should add `getEmojiMap` as one of the earlier arguments like
```javascript
return createSelector(
    (state, post) => post,
    // ...
    getEmojiMap,
    (post, author, reactions, isFlagged, emojiMap) => {
        return (intl) => createAriaLabelForPost(post, author, isFlagged, reactions, emojiMap, intl);
    }
);
```